Implementation/FindMyBLEDevice/FindMyBLEDevice/Views/NewItemPage.xaml

```diff
@@ -15,7 +15,7 @@
     <ContentPage.Content>
         <StackLayout Background="{StaticResource DarkGradient}" Spacing="3" Padding="15">
             <Label Text="User-Label" FontSize="Medium" TextColor="White" />
-            <Entry Text="{Binding UserLabel, Mode=TwoWay}" Placeholder="{Binding Device.AdvertisedName}" FontSize="Medium" TextColor="White" />
+            <Entry Text="{Binding UserLabel, Mode=TwoWay}" Placeholder="{Binding Device.AdvertisedName}" FontSize="Medium" TextColor="White" PlaceholderColor="{StaticResource Accent}"/>
```

Comment:
Since the text color is statically white, independently of primary/secondary/accent colors, I would also assign the placeholder color statically, most likely "Gray" or "LightGray".